decentraland/unity-renderer
# Chore: move assets from resources folder to addressables

## What does this PR change?
Closes #4150. 
Takes out 3.14 MB of `unity.data`: before PR - 57.29 MB, after PR - 54.15 MB
Reduced uncompressed textures from 120 Mb to 58 Mb

- moved common assets from Resources to Addressables : `LoadingHUD`, `LoadingScreen`, `SignupHUD`, `QuestTrackerHUD`, `AirdropHUD`
- moved desktop assets Resources to Addressables: `LoadingHUDDesktop`, `Preloading`, `LoginHUD`
- moved assets in tests Resources to AssetDatabase
- removed BuilderScene from the buildSettings and deleted it from the Assets

## Note 
Other HUDs are extracted to [this ticket](https://app.zenhub.com/workspaces/workstream-core-experience-63aaac346e1c8d00104586e8/issues/gh/decentraland/unity-renderer/4280) 

## How to test the changes?
From the link https://play.decentraland.org/?explorer-branch=chore/move-assets-from-resources-folder-to-addressables

Ideally, it would be nice to check that respective UI and HUDs are working fine
- Loading UI (Web and Desktop)
- Signup UI and Login UI
- QuestTracker
- Airdrop

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
